smtp.go

```diff
@@ -49,7 +49,9 @@ func (m *SMTPEmailBackend) Init(conf map[string]string) error {
 	if err != nil {
 		return err
 	}
-
+	if port == "" {
+		port = conf["SMTPPort"]
```

Comment:
can we change it to:

```
if port == "" {
        port, ok := conf["SMTPPort"]
	if !ok {
		errors.New("SmtpEmailBackend requires specific port config")
	}
```

or setup a default smtp port and add an info message